Packs/DSPM/Integrations/DSPM/DSPM.py

```diff
@@ -0,0 +1,996 @@
+        findings = dspm_get_risk_findings(client, args, page)
+        if not findings:
+            if page == 0:
+                demisto.info("No risks were fetched")
+                readable_output = tableToMarkdown("Risk Findings", [], headers=headers, headerTransform=pascalToSpace)
+
+                return_results(
+                    CommandResults(
+                        outputs_prefix="DSPM.RiskFindings",
+                        outputs_key_field="id",
+                        readable_output=readable_output,
+                    )
+                )
+            break  # No more findings to fetch
+
+        readable_output = tableToMarkdown("Risk Findings", findings, headers=headers, headerTransform=pascalToSpace)
+        return_results(
+            CommandResults(
+                outputs_prefix="DSPM.RiskFindings",
+                outputs_key_field="id",
+                outputs=findings,
+                readable_output=readable_output,
+            )
+        )
+        page += 1
+
+
+def dspm_get_list_of_assets_command(client, args):
+    page = 0
+    headers = ["ID", "Project ID", "Project Name", "Name", "Cloud Provider", "Cloud Environment", "Service Type",
+               "Lifecycle", "Open Risks Count", "Open Alerts Count", "Encrypted", "Open To World",
+               "Tags", "Asset Dig Tags"]
+    while True:
+        assets = get_list_of_assets(client, args, page)
+        if not assets:
+            if page == 0:
+                readable_output = tableToMarkdown("List of assets", [], headers=headers, headerTransform=pascalToSpace)
+
+                return_results(
+                    CommandResults(
+                        outputs_prefix="DSPM.Assets",
+                        outputs_key_field="id",
+                        outputs=[],
+                        readable_output=readable_output,
+                    )
+                )
+            break
+
+        readable_output = tableToMarkdown("List of assets", assets, headers=headers, headerTransform=pascalToSpace)
+
+        return_results(
+            CommandResults(
+                outputs_prefix="DSPM.Assets",
+                outputs_key_field="id",
+                outputs=assets,
+                readable_output=readable_output,
+            )
+        )
+        page += 1
+
+
+def dspm_get_data_types_findings_command(client, args):
+    page = 0
+    all_data_type_findings = []
+    headers = ['No', 'Key']
+
+    while True:
+        data_type_findings = get_data_type_findings(
+            client, args, page
+        )
+
+        if not data_type_findings:
+            if page == 0:
+                readable_output = tableToMarkdown("Data Types", [], headers=headers)
+                return_results(
+                    CommandResults(
+                        outputs_prefix="DSPM.DataTypesFindings",
+                        outputs_key_field="Key",
+                        outputs=[],
+                        readable_output=readable_output,
```

Comment:
```suggestion
                return_results(
                    CommandResults(
                        readable_output=readable_output,
```
in this case we need only the readable_output

Reply:
Sure, we will remove the rest of the parameters.